ff4j-spring-services/pom.xml

```diff
@@ -59,22 +59,22 @@
     <dependency>
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.skyscreamer</groupId>
       <artifactId>jsonassert</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.easymock</groupId>
-      <artifactId>easymock</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-test</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.meanbean</groupId>
       <artifactId>meanbean</artifactId>
+      <scope>test</scope>
```

Comment:
[Already in test scope](https://github.com/ff4j/ff4j-spring-boot-starter-parent/blob/master/pom.xml#L199)